msalautomationapp/src/androidTest/java/com/microsoft/identity/client/msal/automationapp/testpass/broker/mwpj/TestCase2563668.kt

```diff
@@ -0,0 +1,145 @@
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions :
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+package com.microsoft.identity.client.msal.automationapp.testpass.broker.mwpj
+
+import com.microsoft.identity.client.msal.automationapp.R
+import com.microsoft.identity.client.msal.automationapp.testpass.broker.AbstractMsalBrokerTest
+import com.microsoft.identity.client.ui.automation.annotations.LocalBrokerHostDebugUiTest
+import com.microsoft.identity.client.ui.automation.annotations.SupportedBrokers
+import com.microsoft.identity.client.ui.automation.broker.BrokerHost
+import com.microsoft.identity.client.ui.automation.rules.LoadLabUserTestRule
+import com.microsoft.identity.labapi.utilities.client.ILabAccount
+import com.microsoft.identity.labapi.utilities.client.LabQuery
+import com.microsoft.identity.labapi.utilities.constants.AzureEnvironment
+import com.microsoft.identity.labapi.utilities.constants.TempUserType
+import com.microsoft.identity.labapi.utilities.constants.UserType
+import com.microsoft.identity.labapi.utilities.jwt.JWTParserFactory
+import org.junit.Assert
+import org.junit.Before
+import org.junit.Rule
+import org.junit.Test
+import org.junit.rules.TestRule
+
+// https://identitydivision.visualstudio.com/Engineering/_workitems/edit/2563668
+@SupportedBrokers(brokers = [BrokerHost::class])
+@LocalBrokerHostDebugUiTest
+class TestCase2563668 : AbstractMsalBrokerTest() {
+    private lateinit var mUsGovLabAccount: ILabAccount
+
+    @get:Rule
+    val loadAdditionalLabUserRule: TestRule = LoadLabUserTestRule(getAdditionalLabQuery())
+
+    @Test
+    fun test_2563668() {
+        val username = mLabAccount.username
+        val password = mLabAccount.password
+        val usGovUsername = mUsGovLabAccount.username
+        val usGovPassword = mUsGovLabAccount.password
+        val brokerHostApp = broker as BrokerHost
+        brokerHostApp.disableMultipleWpj()
+        // Register tenant A with MWPJ API
+        brokerHostApp.performDeviceRegistrationMultiple(username, password)
+        // Try to register tenant B with MWPJ API, This should fail because broker has MWPJ disabled
+        val errorMessage = brokerHostApp.performDeviceRegistrationMultipleDontValidate(usGovUsername, usGovPassword)
```

Comment:
can we add this step to test case in ado

Reply:
done 79ed44ed0cb3060fe9ad4fcecdfb5c289d55e546